pages/implementation/reference/javascript.mdx

```diff
@@ -15,13 +15,13 @@ Avo can generate JavaScript code targeted at the following platforms
 
 ### Quickstart
 
-Avo functions usage consists of 3 steps.
+Avo functions usage consists of 4 steps.
 
 #### Step 1. Include the Avo file
 
-##### Avo CLI is available through npm
+##### Pull the generated code with the Avo CLI
 
-To install it and get the generated Avo file run the following commands.
+To get the Avo generated JavaScript file you must be a member of an Avo workspace with a JavaScript source. Ask for an invite from a colleague or create a [new workspace](link to https://avo.app/onboarding)
```

Comment:
The link in my suggestion wasn't ready for prime time 😬  I think we need a <Link> component here and stuff right?

Reply:
Looks like native `md` link just works like this [new workspace](https://avo.app/onboarding)

Reply:
Except for opening in new tab, will switch to href for that